app/src/debug/java/io/github/droidkaigi/confsched2017/fps/FpsMeasure.java

```diff
@@ -0,0 +1,26 @@
+package io.github.droidkaigi.confsched2017.fps;
```

Comment:
I think `fps` is too limited naming.
How about `io.github.droidkaigi.confsched2017.debug` package? 🙋 
It's just suggestion! Actually I didn't have strong opinion.